instrumentation/nginx/CMakeLists.txt

```diff
@@ -44,4 +45,5 @@ target_link_libraries(otel_ngx_module
   ${OPENTELEMETRY_CPP_LIBRARIES}
   gRPC::grpc++
   CURL::libcurl
+  nlohmann_json::nlohmann_json
```

Comment:
This is coming from https://github.com/open-telemetry/opentelemetry-cpp/pull/810, not sure if we should attempt to drop this as `OTLP/HTTP+JSON exporter` isn't used at the moment.

Reply:
Does it build when not using `find_package(nlohmann_json)` and not linking to it?

Reply:
Nope :-(

Reply:
Otel CPP external dependencies are starting to get a bit out of hand, wish it had none and would be distributed as a single library file. :smile: 

Anyway, thanks for the PR!